Customizing/README.md

```diff
@@ -4,70 +4,90 @@
 
 1. [Introduction](#introduction)
 1. [User Agreements](#user-agreements)
+   1. [Global user agreements:](#global-user-agreements)
+   1. [Client specific user agreements:](#client-specific-user-agreements)
 1. [System Language Changes](#system-language-changes)
 1. [Skins and Styles](#skins-and-styles)
+1. [Plugins](#plugins)
 
 <!-- /MarkdownTOC -->
 
 <a name="introduction"></a>
-# Introduction
+## Introduction
 
 This directory holds all customized files for this ilias installation.
```

Comment:
It would be great if we always write ILIAS in capital letters, thanks!

Reply:
You are of course right. I had not expected such an "error" and simply overlooked it. I'll amend the pull request.